products/design-system-forms-library/products/components/va-file-input/design-decision-log.md

```diff
@@ -15,15 +15,15 @@ Last updated: 07/17/2025
 
 ### Decision
-The `va-file-input` web component will support setting the `encypted` parameter to `true` when the user needs to supply a password. This will display a password field. This password can be retrieved through the `vaPasswordChange` event. All additional functionality related to decrypting the file or using the provided password will need to be handled by the individual team.
+The `va-file-input` web component will support setting the `encrypted` property to `true` when the user needs to supply a password. This will display a password field. This password can be retrieved through the `vaPasswordChange` event. All additional functionality related to decrypting the file or using the provided password will need to be handled by the individual team.
 
-This decision has been made because of a philisophical approach to building web components that strives to keep the web component as "dry" as possible and to decouple business requirements from the standardized UI. Each team will have a different approach and use case for how they need to handle encrypted and password protected files and the web component should not try to create a single solution for all of those situations. Our team has performed research that revealed there are many unique ways to handle encryption, which reinforces the previous notion. Additionally, the majority of the file related logic should live server side, which is where the file will likely be stored and processed.
+This decision has been made because of a philisophical principle to build components that strives to keep the web component as "DRY" as possible and to decouple business requirements from the standardized UI. Each integration could potentially have a different approach and use case for how they need to handle encrypted and password protected files and the web component should not try to create a single solution for all of those situations. Our team performed research that revealed there are many unique ways to handle encryption. This principle also promotes modularity and allows different parts of an application to be developed, tested, and updated independently. 
 
-We will provide some standardized functions that live in `vets-website`, primarily for usage in forms, that give a basic implementation of how to handle encryption and password protected files.
+Additionally, the majority of file related logic should live server side, which is where the file will likely be stored and processed.
 
+We will provide some standardized utility functions that live in `vets-website`, primarily for usage in forms, that give a basic implementation of how to handle encryption and password protected files.
```

Comment:
> We will provide

@ndsprinkle Should this instead say "Platform will provide..."

Reply:
It could. I was mainly referring to the pattern work in the forms library that has the functionality. But outside of that, it would probably be platform.